src/stores/account.ts

```diff
@@ -72,26 +74,95 @@ export default class AccountStore {
   @observable recentTransactions: TransactionsStore;
   @observable selected: boolean = false;
 
+  /**
+   * Data is potentially dirty and need to be re-downloaded from the server.
+   * This happens after being offline.
+   */
+  set isDirty(value: boolean) {
+    if (value) {
+      runInAction(() => {
+        // mark the recent transactions as dirty
+        this.recentTransactions.isDirty = true;
+        this.recentTransactions.isLoading = false;
+      });
+    }
+    this.isDirty_ = true;
+  }
+
+  get isDirty() {
+    return this.isDirty_;
+  }
+
+  protected isDirty_ = false;
```

Comment:
Isn't the convention to name non-public variables `_isDirty` ?

Reply:
Linter doesnt complain. Its a `protected` field and starting a name with a letter makes it sortable.